zerver/lib/outgoing_webhook.py

```diff
@@ -61,6 +61,8 @@ def make_request(self, base_url: str, event: dict[str, Any], realm: Realm) -> Re
             event["message"],
             apply_markdown=False,
             client_gravatar=False,
+            # TODO: Discuss in CZO.
+            allow_empty_topic_name=False,
```

Comment:
I think we probably want `True` here -- we have no reason to believe outgoing webhooks would fail to process `""` as a topic, and it seems better to keep things simple for them. We do need to be careful to make sure `""` round-trips properly without getting converted to "general chat" for outgoing webhook bots that want to trigger a reply, but that should be OK.

Reply:
Used `True`